ygen/codegen.go

```diff
@@ -87,6 +87,18 @@ type GoOpts struct {
 	// produced if the GenerateJSONSchema YANGCodeGenerator field is set to
 	// true.
 	SchemaVarName string
+	// GoyangImportPath specifies the path that should be used in the generated
+	// code for importing the goyang/pkg/yang package. If it is not specified
```

Comment:
I meant that I would suggest deleting "If not specified it is imported from github.com/openconfig/ygot/ygot." - because it's describing in some behavior that's not related to this structure (how this is initialized should be described in the function where it's actually set to that value) and also introduces a redundancy.